Makefile

```diff
@@ -85,6 +85,8 @@ smoke-test:
 	@md5sum ./build/test.hex
 	tinygo build -size short -o ./build/test.hex -target=circuitplay-express ./examples/thermistor/main.go
 	@md5sum ./build/test.hex
+	tinygo build -size short -o ./build/test.hex -target=pyportal ./examples/touch/resistive/fourwire/main.go
+	@md5sum ./build/test.hex
```

Comment:
The PyPortal example should also be included here in the smoke tests.

Reply:
Will do

Reply:
This has been addressed in the latest set of commits.